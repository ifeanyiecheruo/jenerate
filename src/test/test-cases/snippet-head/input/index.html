<!DOCTYPE html>
<html lang="en-US">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <link rel="x-jen-snippet" href="./head.snippet.html">
    </head>
    <body>
        <p>This is a snippet head test where an x-jen-snippet link is used to pull in snippets.</p>
    </body>
</html>
