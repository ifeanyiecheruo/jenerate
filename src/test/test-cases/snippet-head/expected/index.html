<!DOCTYPE html>
<html lang="en-US">

<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Simple snippet test</title>
</head>

<body>
    <p>This is a snippet head test where an x-jen-snippet link is used to pull in snippets.</p>
</body>

</html>